README.md

```diff
@@ -2,52 +2,55 @@
 
 [![stable](http://badges.github.io/stability-badges/dist/stable.svg)](http://github.com/badges/stability-badges)
 
-Scaffolds [standard](https://github.com/feross/standard) style for a new module. 
+Sets up [standard](https://github.com/feross/standard) style.
 
-- runs `npm install --save-dev standard`
-- adds `standard` to your `"scripts"` test
-
-PRs/suggestions welcome.
+## Usage
 
-<img src="http://i.imgur.com/sdfTwLC.png" width="80%" />
+```console
+$ npm -g install standardize
+$ cd non/standard/project
+$ standardize
+```
 
-## Install
+In `package.json`, the following is prepended to the `test` npm script: `npm run lint &&`.
 
-```sh
-npm install standardize -g
-```
+Then, the project is linted with Standard, and:
 
-## Example
+### If passed:
 
-Just run `standardize` on your module directory
+- installs and saved as dev-dependency: `standard`
+- in `package.json`:
+  - adds an npm script: `"lint": "standard"`
+  - prepends to `test` npm script: `npm run lint`
+- undoes what it did when Standard did not pass (below)
 
-```sh
-standardize
-```
+### If did not pass:
 
-It will auto-install `standard` (if it isn't already installed), then add a new script field that looks like:
+- installs and saves as a dev-dependency:
+  - `eslint`
+  - `eslint-config-standard` and its peer-dependencies
+- sets up an `.eslintrc.json` that:
+  - extends `eslint-config-standard`
+  - excludes all of the rules that didn’t pass
+- in `package.json` adds an npm script `"lint": "eslint **/*.js"`
```

Comment:
Actually, the default patterns include `**/*.jsx`, as well.
https://github.com/Flet/standard-engine/blob/v5.3.0/index.js#L11-L14